### PR TITLE
Team02 - PersonalSchedule Table Components

### PR DESCRIPTION
**Pull request by AlexLaiUCSB**
# Table Component in Storybook
image

### CODE REVIEW

**Review by tiny-babies — APPROVED** [review]
LGTM - nick - checked PR locally.

**Review by ScottUCSB — DISMISSED** [review]
LGTM- Not a whole lot I can add on this review. These were solid adaptions from Team03. It actually reminded me to include the util test for my own task. Well done.

File: frontend/src/fixtures/personalScheduleFixtures.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            "fullName": "Phill Conrad",
+            "givenName": "Phill",
+            "familyName": "Conrad",
+            "emailVerified": true,
+            "locale": "en",
+            "hostedDomain": "ucsb.edu",
+            "admin": true
+          },
+          "description": "Advanced Applications Programming",
+          "quarter": "Winter",
```
Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:20]: Quarter should be stored in the format YYYYQ, always.

Also, quarter should contain the year.

Please consider replacing "Winter" with "20221" for example.

File: frontend/src/fixtures/personalScheduleFixtures.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            "fullName": "Phill Conrad",
+            "givenName": "Phill",
+            "familyName": "Conrad",
+            "emailVerified": true,
+            "locale": "en",
+            "hostedDomain": "ucsb.edu",
+            "admin": true
+          },
+          "description": "Operating Systems",
+          "quarter": "Winter",
```
Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:39]: Same comment as above about YYYYQ

Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:39]: Same comment as above about YYYYQ

File: frontend/src/fixtures/personalScheduleFixtures.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            "fullName": "Phill Conrad",
+            "givenName": "Phill",
+            "familyName": "Conrad",
+            "emailVerified": true,
+            "locale": "en",
+            "hostedDomain": "ucsb.edu",
+            "admin": true
+          },
+          "description": "DATA STRUCT ALGOR",
+          "quarter": "Winter",
```
Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:58]: Same as above about YYYYQ

Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:58]: Same as above about YYYYQ

**Review by pconrad — DISMISSED** [review]
I would suggest a small change to the fixtures that will help in future work, but other than that looks good.

If you make the suggested change, I'm ok to approve this for 10 points.

**Comment by pconrad** [comment]
I'll approve this as is... but, one thing I'd suggest is this... now that you have the correct fixture format, `YYYYQ`, the next thing to do would be to display the quarter in the correct way for end users, i.e. like F20, W22, S21, etc.

The reason for the `YYYYQ` internally is that it sorts properly, with Winter, Spring, the Summer then Fall in the correct order.

There are utiliites here: https://github.com/ucsb-cs156-w22/team04-w22-5pm-courses/blob/main/frontend/src/main/utils/quarterUtilities.js that you can use to make the quarter appear correctly in this table.   I'll add another 10 point issue for fixing this.

**Comment by pconrad** [comment]
Please consider doing this issue next to improve this table component further:

https://github.com/ucsb-cs156-w22/team04-w22-5pm-courses/issues/49

**Review by pconrad — APPROVED** [review]
LGTM.  

I've added an issue to fix something that could be improved (that's a 10 point issue)

**Pull request merged**